VERSIONING.md

```diff
@@ -1,31 +1,58 @@
-# Splunk Distribution of OpenTelemetry Java Instrumentation Versioning
+# Splunk Distribution of OpenTelemetry Java Versioning
+
+Splunk Distribution of OpenTelemetry Java pulls in the the OpenTelemetry Java
```

Comment:
Are you sure that "OpenTelemetry Java" is better in this context than "OpenTelemetry Java Instrumentation"

Reply:
Right now, Splunk standardized on Splunk Distribution of OpenTelemetry <language>. Given both auto and manual is possible likely OK.